alerting/src/test/kotlin/org/opensearch/alerting/MonitorDataSourcesIT.kt

```diff
@@ -209,6 +209,86 @@ class MonitorDataSourcesIT : AlertingSingleNodeTestCase() {
         assertEquals("Didn't match all 8 queries", 8, findings[0].docLevelQueries.size)
     }
 
+    fun `test execute monitor with nested json doc as source`() {
```

Comment:
let's not call it nested.. let's say object type

nested is an opensearch field mapping terminlogy
let's avoid that confusion